notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>Also instead of saying "charts similar to the screenshot" - perhaps explain that the dashboard has 3 sections, the top for Lambda Metrics, the second for SageMaker metrics, and the bottom for Alarms which are pictured below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='107'/>